card/mod/carrierwave/Gemfile

```diff
@@ -0,0 +1,4 @@
+# the current release 0.11.2 uses an old version of the fog gem which isn't
+# compatible with rails 4.2
+gem "carrierwave", git: "https://github.com/carrierwaveuploader/carrierwave",
+                   branch: "master"
```

Comment:
I forgot that I implemented an alternative load procedure for the gem. I added carrierwave as submodule and added the submodule directory to the load path. Then I can just require it:
```ruby
cw_path = File.expand_path "../../vendor/carrierwave/lib", __FILE__
$LOAD_PATH.unshift(cw_path) unless $LOAD_PATH.include?(cw_path)
require "carrierwave"
```